### ISSUE TITLE
No Config.ini file

### ISSUE THREAD

**Issue by Sarin-jacob** [purpledalek/SimplePiStats#1]
After running the install.sh script no config.ini file was created. 
Failed to start SimplePiStats.service
```
Traceback (most recent call last):
  File "/home/pi/pistats/SimplePiStats/SimplePiStats.py", line 30, in <module>
    port = conf_get("port")
           ^^^^^^^^^^^^^^^^
  File "/home/pi/pistats/SimplePiStats/SimplePiStats.py", line 23, in conf_get
    return ast.literal_eval(config.get("SimplePiStats", option))
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/configparser.py", line 797, in get
    d = self._unify_values(section, vars)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/configparser.py", line 1168, in _unify_values
    raise NoSectionError(section) from None
configparser.NoSectionError: No section: 'SimplePiStats'
SimplePiStats.service: Main process exited, code=exited, status=1/FAILURE
```

**Comment by purpledalek** [comment]
I'm aware of this issue and I know what the problem is, a fix will be pushed later today

**Comment by purpledalek** [comment]
A fix for this has been implemented, if there's any more issues please let me know. Hope you like SimplePiStats!

**Comment by Sarin-jacob** [comment]
Is there any way to set the listening address easily?
 I believe rn its defaults to 0.0.0.0:5555.
It would be better for security, if it could be restricted.

The page takes certain time to load (after collecting the results of du(disk usage), It would be better to load the page with previous values, then populate it afterwards.

The webpage is mostly responsive, bt the settings overlay isnt, As of right now the settings cannot be changed without going into desktop site

Any plans to add a custom css option?

The project looks nice and I like it, This makes doing small things so much easier.

**Comment by purpledalek** [comment]
The listening address will be moved to the config in the next update along with the settings bug you mentioned being fixed. i will look into getting disk usage values after page load too, if i can manage it, that'll also be in the update. in regards to custom CSS, were you thinking of more options to be added to settings (like font style, size, color), or the ability to point to a custom CSS file?

**Comment by Sarin-jacob** [comment]
By custom css, I mean a setting in which users can add css. No need to make an input for every css element or class.
Like this.. 
![custom-css-customcssfield-e089d319e00e4efe44fbaf11aef9aec6.png](https://github.com/purpledalek/SimplePiStats/assets/53591235/1440ad76-702d-4a08-887d-f6dcb7e51b9e)

In the docs you can add any custom classes or tags used.
I saw this feature in jellyfin - [Custom Css](https://jellyfin.org/docs/general/clients/css-customization/)
 If its done like this then everyone can customize the looks of the page freely.
> or the ability to point to a custom CSS file?

Pointing to a custom css would be fine I think, bt then again wouldn't it be better to add the content of custom css to the end of the current css file? So that if only one or two tweaks are to made user can add styles just for those elements not worrying about others..

**Comment by purpledalek** [comment]
just pushed an update with the changes you suggested!

**Comment by purpledalek** [comment]
sidenote; I'd recommend copying the config.ini folder somewhere else if you've already customised it, have it load in once, then open the config.ini file and add what you've edited into their resspective sections, simply due to the fact that the listen address won't exist in your current config.ini file

**Comment by Sarin-jacob** [comment]
Yeah I had it backed up. Now it is loading in a snap. I have tried the custom CSS n its working nicely, I can edit from mobile too.  Nice work! :) 
With the ability to change the listening address, No more worrying about someone messing with the services n pi. I changed mine to tailscale IP, so now its only available to my devices.

I have some more feature requests though some are not easy to implement I think. should I open a new issue for those? I should close :)
